assets/javascripts/omise-payment-form-handler.js

```diff
@@ -202,15 +202,63 @@
 				opacity: 0.6
 			}
 		});
-		const billingAddress = {
-			country: document.getElementById('billing_country').value,
-			postal_code: document.getElementById('billing_postcode').value,
-			state: document.getElementById('billing_state').value,
-			city: document.getElementById('billing_city').value,
-			street1: document.getElementById('billing_address_1').value,
-			street2: document.getElementById('billing_address_2').value,
+
+		const billingAddress = getBillingAddress();
+		OmiseCard.requestCardToken(billingAddress);
+	}
+
+	/**
+	 * @returns object | null
+	 */
+	function getBillingAddress() {
+		const billingAddress = {};
+		const billingAddressFields = [
```

Comment:
Isn't this more naturally a dictionary? e.g.

```
{
  country: "billing_country",
  postal_code: "billing_postcode",
  ...
}
```

Reply:
Yes. Thank you.